osquery/tables/applications/browser_chrome.cpp

```diff
@@ -30,7 +32,30 @@ QueryData genChromeExtensions(QueryContext& context) {
     chromePath = "/.config/google-chrome/%/Extensions/";
   }
 
-  return genChromeBasedExtensions(context, chromePath);
+  fs::path bravePath;
+
+  if (isPlatform(PlatformType::TYPE_WINDOWS)) {
+    bravePath = "\\AppData\\Roaming\\brave\\Extensions\\";
```

Comment:
New string literals like that should be moved at the top of the file inside an anonymous namespace

Example:
```
namespace {
static const std::string kBravePath{"\\AppData\\Roaming\\brave\\Extensions\\"};
}
```